### DISCUSSION TITLE
Actividad 0 - Martinez, Lucas

### DISCUSSION THREAD

**Discussion by lucasmartinez22** [frlp-utn-ingsoft/catedra#75] [Show and tell]
https://github.com/lucasmartinez22/Actividad-0---Martinez-Lucas.git

**Comment by RodrigoJacznik** [comment]
Buenas,
Los commits estan bien.
La app parece funcionar bien pero intente votar en la encuesta que esta cargada y no funciona. A vos te funcionó?

Para aprobar la actividad la app tiene que funcionar.

**Reply by lucasmartinez22** [reply]
Buenas, es extraño porque cuando la probé el finde funcionaba las respuestas. Ahora quise responder y no me dejó. Reviso y te aviso.

**Comment by lucasmartinez22** [comment]
@RodrigoJacznik profesor ya corregí el codigo, tenia un error en una linea. Avisame si quedo correctamente cargado ahora.

**Comment by RodrigoJacznik** [comment]
Hola, ahora hay otro error en el index
En este commit: https://github.com/lucasmartinez22/Actividad-0---Martinez-Lucas/commit/4f84d7c06953f96c8e7081769a08974d6162c999 cambiaste el template `index.html` y ahora rompe. Como estaba antes creo que esta correcto

**Reply by lucasmartinez22** [reply]
@RodrigoJacznik correcto. Gracias! corregido. Perdón, no me habia dado cuenta de testear el index. Ahora ya testie todo. Espero su respuesta

**Comment by lucasmartinez22** [comment]
corregido y subido!

**Reply by ivanduva** [reply]
Perfecto Lucas, queda aprobado.
Igualmente fijate que dejaste creada una poll vacía, seguramente lo hiciste desde el admin y te deja hacerlo sin preguntas. Pero la otra poll quedó bien.

Saludos!
